### PR TITLE
theming: IR specific view

### PR DESCRIPTION
**Pull request by sebastiendeleze**
* NEW Change layout in IR context

Signed-off-by: Sébastien Délèze <sebastien.deleze@rero.ch>

### CODE REVIEW

File: sonar/theme/assets/js/config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import angular from 'angular'
+
+angular.module('invenioConfig', [])
+  .controller('configController', ['$scope', '$log', '$http', '$sce', function($scope, $log, $http, $sce) {
```
Comment (jma) [inline sonar/theme/assets/js/config.js]: perhaps a small comment...

File: sonar/theme/assets/scss/sonar/theme.scss [hunk 2]
```diff
@@ -1,16 +1,3 @@
-/*
```
Comment (jma) [inline sonar/theme/assets/scss/sonar/theme.scss]: please add your own licence

File: sonar/theme/assets/scss/usi/theme.scss [hunk 3]
```diff
@@ -1,21 +1,3 @@
-/*
```
Comment (jma) [inline sonar/theme/assets/scss/usi/theme.scss]: please add your own licence

File: sonar/theme/assets/scss/sonar/theme.scss (hunk 2 above)
Comment (sebastiendeleze) [reply sonar/theme/assets/scss/sonar/theme.scss]: I erased license by error.. I noticed that several files don't have license, I will do a PR for that and for commenting files, too.

**Pull request merged**